entity/template.go

```diff
@@ -0,0 +1,15 @@
+package entity
+
+type Template struct {
+	Text    string   `json:"text"`
+	Href    string   `json:"href"`
+	Plugins []string `json:"plugins"`
+	Desc    string   `json:"desc"`
+	Tags    []string `json:"tags"`
+	Icon    string   `json:"icon"`
+	Value   string   `json:"value"`
+	EnvVars []struct {
+		Name string `json:"name"`
+		Desc string `json:"desc"`
```

Comment:
Similar to above comment, can have optional and default here. The type we use on the frontend is

![image](https://user-images.githubusercontent.com/3044853/112528807-7c9cd680-8d9c-11eb-8e5e-59ff4cdf729a.png)